reporthandling/helpers/v1/cloudmetadata.go

```diff
@@ -106,22 +106,24 @@ func (eks EKSMetadata) join(s []string) string {
 // TODO: implement AKS parser support
 
 type AKSMetadata struct {
-	// name string
+	name string
 }
 
-func NewAKSMetadata() *AKSMetadata {
-	return &AKSMetadata{}
+func NewAKSMetadata(name string) *AKSMetadata {
+	return &AKSMetadata{
+		name: name,
+	}
 }
 
 // GetName returns the full name of the cluster
-func (aks AKSMetadata) GetName() string {
-	return ""
+func (aks *AKSMetadata) GetName() string {
+	return aks.name
 }
 
 func (aks AKSMetadata) Provider() apis.CloudProviderName {
```

Comment:
I would change this to pointer receiver as well

Reply:
What do you mean?